pyvista/plotting/picking.py

```diff
@@ -1157,13 +1157,12 @@ def disable_picking(self):
         >>> pl.disable_picking()
 
         """
-        if self._picker is not None:
+        if getattr(self, '_picker', None):
             self._picker.RemoveObservers(_vtk.vtkCommand.EndPickEvent)
 
         # remove left clicking observer if available
-        if hasattr(self, 'iren'):
-            if self.iren is not None:
-                self.iren.remove_observer(self._picking_left_clicking_observer)
+        if getattr(self, 'iren', None):
+            self.iren.remove_observer(self._picking_left_clicking_observer)
```

Comment:
Since this was the same logic I changed this to how I implemented the new check. If this is too smart then I'll change back both the original, two-step check. I can't tell how obscure my version is.

Reply:
This works. Didn’t realize that `getattr` allowed you to have a default return. Since it’s like `Dict.get`, it’s intuitive enough.